actions/check-copyrights/action.yml

```diff
@@ -25,6 +25,13 @@ inputs:
     required: false
     default: https://www.apache.org/dyn/closer.cgi?action=download&filename=
     # Another alternative: http://www.mirrorservice.org/sites/ftp.apache.org/
+  xml_file:
```

Comment:
I'm not sure if `xml_file` is a helpful name; *lots* of things are XML files. Maybe `config` would be more informative? Otherwise the semantics of it look fine.

I think the "supported options" text could be put within the description.